GitCommands/ExternalLinks/ExternalLinkDefinition.cs

```diff
@@ -38,7 +38,7 @@ public enum RemotePart
         /// <summary>
         /// List of formats to be applied for each revision part matched by SearchPattern
         /// </summary>
-        public BindingList<ExternalLinkFormat> LinkFormats = new BindingList<ExternalLinkFormat>();
+        public BindingList<ExternalLinkFormat> LinkFormats { get; } = new BindingList<ExternalLinkFormat>();
```

Comment:
💭 The title of this commit is somewhat misleading. This is more a field&rarr;property conversion than simply marking fields `readonly`.